packages/main/src/components/AnalyticalTable/AnayticalTable.jss.ts

```diff
@@ -12,7 +12,10 @@ const styles = {
     minHeight: '3rem',
     fontFamily: ThemingParameters.sapFontFamily,
     fontSize: ThemingParameters.sapFontSize,
-    fontWeight: 'normal'
+    fontWeight: 'normal',
+    '&::-webkit-scrollbar': {
+      height: ThemingParameters.sapScrollBar_Dimension
+    }
```

Comment:
```suggestion
    '&::-webkit-scrollbar': {
      height: ThemingParameters.sapScrollBar_Dimension
    },
    '&::-webkit-scrollbar-thumb': {
      height: ThemingParameters.sapScrollBar_Dimension
    }
```
I think the scrollbar thumb should be also styled